packages/flutter/lib/src/widgets/localizations.dart

```diff
@@ -97,6 +99,12 @@ abstract class LocalizationsDelegate<T> {
   /// const constructors so that they can be used in const expressions.
   const LocalizationsDelegate();
 
+  /// Return true if the instance of 'T' loaded by this delegate's [load]
+  /// method supports the given `locale`'s language.
+  bool isSupported(Locale locale) => true;
```

Comment:
maybe prefix with something like:
```
/// Whether resources for the given locale can be loaded by this delegate.
///
```

Also either use backticks or no quotes at all around T. Single quotes makes it look like a string.